packages/dev/core/src/Rendering/renderingManager.ts

```diff
@@ -85,6 +85,12 @@ export class RenderingManager {
     private _customTransparentSortCompareFn: { [id: number]: Nullable<(a: SubMesh, b: SubMesh) => number> } = {};
     private _renderingGroupInfo: Nullable<RenderingGroupInfo> = new RenderingGroupInfo();
 
+    /**
+     * Gets or sets a boolean indicating that the manager will not reset between frames.
+     * This means that if a mesh becomes invisible or transparent it will not be visible until this boolean is set to false again.s
+     */
+    public maintainStateBetweenFrames = false;

```

Comment:
I think you will need a setter that will reset all the `_wasDispatched` properties to `false` when `maintainStateBetweenFrames` is switched from `true` to `false`?

Reply:
yeah this is a good catch! PR is coming!